libstuff/libstuff.cpp

```diff
@@ -2760,6 +2753,18 @@ bool SREMatch(const string& regExp, const string& s, string& match) {
     return pcrecpp::RE(regExp).FullMatch(s, &match);
 }
 
+void redactredactSensitiveValues(string& s) {
```

Comment:
This should probably be called `SRedactSensitiveValues`. It should definitely not be called `redactredactSensitiveValues` 😆

Reply:
I don't know whatwhat you mean. 😆